dataminer-overview/Training_courses_and_certification/Overview_Training_certification.md

```diff
@@ -14,3 +14,101 @@ For example, the DataMiner Fundamentals Certification is intended for DataMiner
 
 > [!TIP]
 > For more information about the different certifications and their process, please visit [DataMiner Certification](https://community.dataminer.services/learning/certification/).
+
+## DataMiner Fundamentals exam guide
+
+Are you preparing to take the DataMiner Fundamentals exam? Below you can find all the information you need, including the certification process, the exam format, the scoring criteria, and tips on how to prepare for the exam.
+
+### Certification process
+
+As soon as the ordering process is finalized and your certification process can be started, you will receive an automated email from Skyline Academy with information about the process and the deadline.
+
+You will also receive an email with your user ID and password to access the training DMA in order to practice and to take the exam.
+
+To book an appointment, you will need to use the booking link that you will find in the email you received with information about the start of the certification process.
+
+> [!NOTE]
+>
+> - The certification process is valid for 3 months, unless otherwise agreed.
```

Comment:
Do you mean that the maximum duration of the certification process is 3 months, unless otherwise agreed?

Reply:
The candidate has max. 3 months to finalize the process (pass the exam incl. retakes). Unless agreed otherwise because some organizations want their employees to pass the exam within 1 month after the process started.